mpsl: cx: Do not support CX with FEM on 53 (#10924)
nrfconnect/sdk-nrf: doc/nrf/app_dev/wifi_coex/index.rst

If your application uses SR protocols and requires coexistence with Wi-Fi, you must enable :ref:`nrfxlib:mpsl_cx` in the :ref:`nrfxlib:mpsl` (MPSL) library.

> On the nRF5340, Wi-Fi Coexistence shall not be enabled in conjunction with FEM, as this can lead to undefined behavior.

Reviewer: ```suggestion
Do not enable Wi-Fi coexistence on the nRF5340 DK in conjunction with FEM, as this can lead to undefined behavior.
```
